0002-data-collection-extension/README.md

```diff
@@ -1,96 +1,129 @@
 ---
-name: Data Collection Extension protocol
+name: Data Collection extension protocol
```

Comment:
I'm fine with `Data Collection Extension`.

As for capitalization, why `Collection` should be capitalized and `extension` should not?

P.S. I am not sure if Kaa documentation guidelines apply to RFCs.